README.md

```diff
@@ -2,15 +2,14 @@
 
 :baby: Staff interface for Kulttuurin kummilapset / Godchildren of Culture :violin:
 
-## Development with Docker
-
-To start the app in a docker container, run:
-
-### `docker-compose up`
-
-Open [http://localhost:3001](http://localhost:3001) to view it in the browser.
-
-When there are changes that need rebuilding the container, run `docker-compose up --build` instead.
+## Getting started
+
+1. Clone the repo.
+2. Use file `.env.local` to modify environment variables if needed. For more info, check [this](https://create-react-app.dev/docs/adding-custom-environment-variables#docsNav).
+3. Run either
+    * `yarn start` to run the app normally **or**
+    * `docker-compose up` to run the app in a Docker container. In the future when there are changes that need rebuilding the container, run `docker-compose up --build` instead.
+4. Open [http://localhost:3001](http://localhost:3001) to view the app in the browser.
 
```

Comment:
Could be nice to add info on how to get this working with a local tunnistamo! 

I created a `.env.development.local` with `
REACT_APP_OIDC_AUTHORITY="http://tunnistamo-backend:8000"` and then executed in Tunnistamo something like this: 
```
./manage.py add_oidc_client -n kukkuu-admin-ui -t "id_token token" -u "http://localhost:3001/callback" "http://localhost:3001/silent_renew" "http://localhost:3001/silent_renew.html" -i https://api.hel.fi/auth/kukkuu-admin-ui -m github -s dev
./manage.py add_oidc_client -n kukkuu-api -t "code" -u http://localhost:8081/return -i https://api.hel.fi/auth/kukkuu -m github -s dev -c
./manage.py add_oidc_api -n kukkuu -d https://api.hel.fi/auth -s email,profile -c https://api.hel.fi/auth/kukkuu
./manage.py add_oidc_api_scope -an kukkuu -c https://api.hel.fi/auth/kukkuu-admin-ui -n "Kulttuurin kummilapset Admin UI" -d"Lorem ipsum"
```

Reply:
Added (well, basically copy-pasted) instructions for setting up a local Tunnistamo written by some fine gentleman.